data/test_domains/default_with_slots_and_no_actions.yml

```diff
@@ -18,7 +18,7 @@ slots:
   name:
     type: text
 
-templates:
+responses:
```

Comment:
leftover from an earlier rename (created a warning that it is deprecated, that is why I changed it)